packages/core/useFetch/index.ts

```diff
@@ -349,7 +349,7 @@ export function useFetch<T>(url: MaybeComputedRef<string>, ...args: any[]): UseF
   const statusCode = ref<number | null>(null)
   const response = shallowRef<Response | null>(null)
   const error = shallowRef<any>(null)
-  const data = shallowRef<T | null>(initialData)
+  const data = shallowRef<T | null>(initialData || null)
```

Comment:
Be careful, if `initialData` is falsy, it will be set as null, not expected.
E.g. if I set `{ initialData: false }` as option, then `data` will be set to `null` instead of `false`. Is it possible to use `??` instead of `||`?